api/frontend.proto

```diff
@@ -84,25 +84,31 @@ message WatchAssignmentsResponse {
 // BETA FEATURE WARNING: This Request message is not finalized and still subject
 // to possible change or removal.
 message AcknowledgeBackfillRequest {
+  // A Backfill ID to acknoledge.
   string backfill_id = 1;
+
+  // An updated Assignment of the requested Backfill.
   Assignment assignment = 2;
 }
 
 // BETA FEATURE WARNING: This Request message is not finalized and still subject
 // to possible change or removal.
 message CreateBackfillRequest {
+  // An empty Backfill object.
   Backfill backfill = 1;
 }
 
 // BETA FEATURE WARNING: This Request message is not finalized and still subject
 // to possible change or removal.
 message DeleteBackfillRequest {
+  // An existing Backfill ID.
```

Comment:
Just a personal preference but `ID of backfill ticket to delete`. The other could follow the same pattern but otherwise it looks good to me.

Reply:
nice, i like it. Will change it